src/main/java/de/hysky/skyblocker/utils/ApiUtils.java

```diff
@@ -43,6 +47,10 @@ public static String name2Uuid(String name) {
 				NAME_2_UUID_CACHE.put(name, uuid);
 
 				return uuid;
+			} else if (response.ratelimited() && retries < 3) {
+				Thread.sleep(800);
```

Comment:
Why `retries < 3` and why sleep for specifically 800 ms?

Reply:
The numbers are from the authlib's game profile repository except for 800 which I had increased just a little bit. This was to offset the rate limits from all the lookups from the custom party finder in case we ever need to look up names in the mod while people are in that GUI.